config/config-sil.json

```diff
@@ -1,5 +1,13 @@
 {
-    "iso15118_charger": {
+    "bsm_power_meter" : {
+        "module": "PowermeterBSM",
+        "config_module": {
+            "power_unit_id" : 42,
+            "update_interval" : 5,
+            "serial_device" : "/dev/ttyUSB0"
```

Comment:
A SIL with a HW dependency is not a SIL. Perhaps a little BSM_Simulator is better here like the slac or imd simulator